packages/sdk/src/extensions/integrations/integrationTypes.ts

```diff
@@ -56,6 +57,8 @@ export const Integration = {
   IdleV4Lend: "IdleV4Lend",
   IdleV4Redeem: "IdleV4Redeem",
   IdleV4ClaimRewards: "IdleV4ClaimRewards",
+  Erc4626Lend: "Erc4626Lend",
+  Erc4626Redeem: "Erc4626Redeem",
```

Comment:
Make that `ERC4626` here and elsewhere please! It's an abbreviation and we should be capitalizing every latter in that case.